packages/elements-react/src/context/form-state.ts

```diff
@@ -46,6 +47,12 @@ function parseStateFromFlow(flow: OryFlowContainer): FormState {
       ) {
         return { current: "method_active", method: flow.flow.active }
       } else if (isChoosingMethod(flow.flow.ui.nodes)) {
+        // Login has a special case where we only have one method. Here, we
+        // do not want to display the chooser.
+        const authMethods = nodesToAuthMethodGroups(flow.flow.ui.nodes)
+        if (authMethods.length === 1) {
+          return { current: "method_active", method: authMethods[0] }
+        }
```

Comment:
This leads to this screen being shown if code is the only option for the user.

And in that case, we either need to auto submit the flow to then immediately show the code input, or _do_ show the method selector so that the user can trigger the code sending. 

<img width="555" alt="image" src="https://github.com/user-attachments/assets/9aed16f5-ed62-4816-9110-13de5af0ccd5" />

Reply:
Fair point! I think auto-submit is the way to go. Ideally, this would be solved in kratos' identifier first

Reply:
Yea, actually I don't think we should work around this in Elements and just fix it in Kratos. So for now, let's show the method selector and fix it properly later in Kratos.

Reply:
But only for the code method please, not for password etc.

Reply:
https://github.com/ory/elements/pull/336